components/mpas-framework/src/driver/mpas.F

```diff
@@ -12,7 +12,6 @@ program mpas
 
    implicit none
 
-   type (core_type), pointer :: corelist => null()
```

Comment:
@mark-petersen, I don't see how this fix can be correct. Where is `corelist` coming from with this change?

Reply:
Sorry, I see it now:
https://github.com/mark-petersen/E3SM/blob/mark-petersen/mpas/standalone-compile-fix/components/mpas-framework/src/driver/mpas_subdriver.F#L41

Reply:
It was added to here on Dec 21:

```
git diff 09491ec080 46b65c1b4f
...
index 33f463b4af..14db2022bf 100644
--- a/components/mpas-framework/src/driver/mpas_subdriver.F
+++ b/components/mpas-framework/src/driver/mpas_subdriver.F
@@ -35,6 +35,12 @@ module mpas_subdriver
    use test_core_interface
 #endif

+#ifdef HAVE_MOAB
+   use mpas_moabmesh
+#endif
+   type (core_type), pointer :: corelist => null()
```
and `mpas.F` has the line:
```
 10    use mpas_subdriver
```